keylime/cloud_verifier_tornado.py

```diff
@@ -859,7 +864,7 @@ async def invoke_provide_v(agent):
         pass
     v_json_message = cloud_verifier_common.prepare_v(agent)
 
-    # TODO: remove special handling after initial upgrade
+    # ssl_context is only None with it mTLS verification was disabled in the configuration
```

Comment:
parser error in comment near '...None with it mTLS ...'; suggested solution: `... None if mTLS ...`

Reply:
Thanks for catching that. Is now fixed.